sonar-slang-plugin/src/main/resources/org/sonar/l10n/kotlin/rules/kotlin/S1066.html

```diff
@@ -0,0 +1 @@
+<p>Merging collapsible <code>if</code> statements increases the code's readability.</p>
```

Comment:
The Compliant / Noncompliant sections are missing. You can create a `Kotlin` subtask in the RSPEC to override these parts